content/security/improvements.adoc

```diff
@@ -0,0 +1,66 @@
+---
+layout: security
+title: "Improvements by the Security Team"
+section: security
+---
+
+These are some contributions by members of the Jenkins security team that weren't delivered as security fixes, but still are security-related.
+
+
+== 2020
+
+Jenkins (core)::
+- Plugin Manager: https://github.com/jenkinsci/jenkins/pull/4553[Add inline security warnings to installed plugins list]
+- Plugin Manager: https://github.com/jenkinsci/jenkins/pull/4513[Show on 'updates' tab when a warning would be fixed]
+- UI/UX: https://github.com/jenkinsci/jenkins/pull/4389[Add stack trace suppression into core as a standard behavior]
+- UI/UX: https://github.com/jenkinsci/jenkins/pull/4482[Do not show disabled permissions in permission errors]
```

Comment:
Optional, and possibly ill-advised to include this, since the links will eventually be broken when those releases are no longer displayed in the changelog

```suggestion
- UI/UX: https://github.com/jenkinsci/jenkins/pull/4482[Do not show disabled permissions in permission errors] - link:/changelog/#v2.221[Jenkins 2.221]
```